test/EVMHost.h

```diff
@@ -86,15 +88,23 @@ class EVMHost: public evmc::Host
 		m_state.logs.clear();
 	}
 
-	bool account_exists(evmc::address const& _addr) noexcept final
+	bool account_exists(evmc::address const& _addr) const noexcept final
 	{
+		// Make all precompiled contracts exist.
```

Comment:
Ok - if we never actually need them in ``m_state.accounts``, but only need this to return true, then this is way nicer than creating them in the constructor :+1:!

Reply:
They should actually exists with a 1 wei balance if we want to mirror what happens on mainnet. (Without a balance, code and state they are considered "empty" and be purged.)

Reply:
Well, this might be subject for another PR.